sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDBFileManager.scala

```diff
@@ -290,8 +290,8 @@ class RocksDBFileManager(
       colFamilyIdMapping, colFamilyTypeMapping, maxColumnFamilyId)
     val metadataFile = localMetadataFile(checkpointDir)
     metadata.writeToFile(metadataFile)
-    logInfo(log"Written metadata for version ${MDC(LogKeys.VERSION_NUM, version)}:\n" +
-      log"${MDC(LogKeys.METADATA_JSON, metadata.prettyJson)}")
+    logInfo(log"Written metadata for version ${MDC(LogKeys.VERSION_NUM, version)}")
```

Comment:
Is this really an issue ? How often did you see this ?

Reply:
@vinodkc - if this is not very verbose, lets revert to original and same for the line below ?

Reply:
Done

Reply:
It was printing the content of metadata.prettyJson, which is verbose,

Reply:
Ok then lets not split the line and move to DEBUG

Reply:
Done